output/typescript/types.ts

```diff
@@ -3631,11 +3632,19 @@ export interface FieldCapabilitiesBodyIndexFilterRangeTimestamp {
   lt?: integer
 }
 
+export interface FieldCapabilitiesBodyIndexFilterTerm {
+  versionControl: FieldCapabilitiesBodyIndexFilterTermVersionControl
```

Comment:
is this really `versionControl`? Might be worth opening a ticket on elasticsearch if so should be `version_control`.